srcs/web/app.py

```diff
@@ -24,6 +27,23 @@
 app.add_route("/metrics", prometheus_metrics)
 
 
+@lru_cache()
+def get_args() -> EnvSettings:
+    return EnvSettings()
+
+
+@lru_cache()
+def get_redis() -> RedisManager:
+    args: EnvSettings = get_args()
+    settings: dict = setup_settings(args.setting_file)
```

Comment:
Why is setup_settings not setup as a `lru_cache`?

Reply:
I find that it made the code look messy. Nevertheless, I made the change. Let me know what you think of it. I've tested everything, and it works perfectly.